de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/SarosSession.java

```diff
@@ -1080,7 +1110,7 @@ private SarosSession(final String id, IContainerContext context,
         context.initComponent(this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "initComponent"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=257&issues=AWbeHvOCSgN4uCclRsHH&open=AWbeHvOCSgN4uCclRsHH)

[ISSUE_KEY]: AWbeHvOCSgN4uCclRsHH